bin/k8s-conjur-deploy/3_init_conjur_cert_authority.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+
+. utils.sh
+
+announce "Initializing Conjur certificate authority."
+
+set_namespace $CONJUR_NAMESPACE_NAME
+
+conjur_master=$(get_master_pod_name)
+
+if [ $CONJUR_VERSION = '4' ]; then
```

Comment:
our feature doesn't support conjur v4. we can remove all instances of it

Reply:
copied file as is. will fix it.